app/public/css/application.css

```diff
@@ -15,7 +15,7 @@ video {
 video::-webkit-media-controls-fullscreen-button {
   display: none;
 }
-
+/* can't test that one locally but this is the place where the decision logo is styled @paulactivecinema */
```

Comment:
I think you can't mention someone from the source code but from inline comments on GitHub @slayerpart. Also I advice you not to write comments out of context. How could you understand this line if you don't know that this is on GitHub?